src/foam/comics/v2/DAOSummaryView.js

```diff
@@ -94,7 +99,14 @@ foam.CLASS({
     {
       name: 'delete',
       code: function() {
-        alert('TODO');
+        var self = this;
+        this.config.dao.remove(this.data).then(function() {
+          self.finished.pub();
+          self.stack.back();
+        }, function(e) {
+          self.exception = e;
+          self.throwError.pub();
```

Comment:
Who is listening for throwError?  It might be useful to add the exception as an argument to pub() so that receivers don't have to go looking for it.

Reply:
Done. Added the exception as an argument to pub() to DAOSummaryView, DAOCreateView and DAOUpdateView.